documentation/source/Usecases/mongodb.rst

```diff
@@ -0,0 +1,431 @@
+
+The benchmark output displayed is similar to the following:
+::
+    sysbench 1.0:  multi-threaded system evaluation benchmark
+
+    Running the test with following options: 
+    Number of threads: 10
+    Report intermediate results every 10 second(s)
+    Initializing random number generator from current time
+
+
+    Initializing worker threads...
+
+    setting write concern to 1
+    Threads started!
+
+    [  10s] threads: 10, tps: 56.60, reads: 171.50, writes: 170.40, response time: 316.14ms (95%), errors: 0.00, reconnects:  0.00
+    [  20s] threads: 10, tps: 74.70, reads: 222.90, writes: 223.50, response time: 196.30ms (95%), errors: 0.00, reconnects:  0.00
+    [  30s] threads: 10, tps: 76.00, reads: 227.70, writes: 228.00, response time: 196.71ms (95%), errors: 0.00, reconnects:  0.00
+    [  40s] threads: 10, tps: 79.60, reads: 239.70, writes: 238.80, response time: 329.08ms (95%), errors: 0.00, reconnects:  0.00
+    :
+    :
+    OLTP test statistics:
+      queries performed:
+          read:                            154189
+          write:                           154122
+          other:                           51374
+          total:                           359685
+      transactions:                        51374  (85.61 per sec.)
+      read/write requests:                 308311 (513.79 per sec.)
+      other operations:                    51374  (85.61 per sec.)
+      ignored errors:                      0      (0.00 per sec.)
+      reconnects:                          0      (0.00 per sec.)
+
+    General statistics:
+        total time:                          600.0703s
+        total number of events:              51374
+        total time taken by event execution: 6000.1853s
+        response time:
+             min:                                 26.11ms
+             avg:                                116.79ms
+             max:                               2388.03ms
+             approx.  95 percentile:             224.00ms
+
+    Threads fairness:
+        events (avg/stddev):           5137.4000/21.50
+        execution time (avg/stddev):   600.0185/0.02
+
+While the benchmark is in progress, performance and capacity usage statistics on the OpenEBS storage volume can be viewed using the mayactl commands that must be executed on the maya-apiserver pod.
+
+Take an interactive bash session into the maya-apiserver pod container.
+::
+  
+    test@Master:~$ kubectl exec -it maya-apiserver-1089964587-x5q15 /bin/bash
+    root@maya-apiserver-1089964587-x5q15:/#
+
+Obtain the list of OpenEBS persistent volumes created by the MongoDB statefulset application YAML.
+::
+
+    root@maya-apiserver-1089964587-x5q15:/# maya volume list
+    Name                                      Status
+    pvc-0d39583c-bad7-11e7-869d-000c298ff5fc  Running
+    pvc-21da76b6-bad7-11e7-869d-000c298ff5fc  Running
+
```

Comment:
```
     root@maya-apiserver-1089964587-x5q15:/# maya volume list
     Name                                      Status
     pvc-0d39583c-bad7-11e7-869d-000c298ff5fc  Running
     pvc-21da76b6-bad7-11e7-869d-000c298ff5fc  Running
      :  
      ```

Reply:
Done!

Reply:
There is a sentence "Take an interactive bash session into the maya-apiserver pod container."
Can we say "include an interactive..." or "run an interactive..."?